include/kanshi.h

```diff
@@ -5,6 +5,9 @@
 #include <wayland-client.h>
 
 struct zwlr_output_manager_v1;
+#ifdef KANSHI_HAS_VARLINK
+typedef struct VarlinkService VarlinkService;
```

Comment:
Why is this necessary?

Reply:
Took this out, it's not necessary and is only for consistency with the typedef that varlink uses.